api/hooks-common.md

```diff
@@ -491,6 +491,90 @@ Populates items *recursively* to any depth. Supports 1:1, 1:n and n:1 relationsh
 - Provides performance profile information.
 - Backward compatible with the old FeathersJS `populate` hook.
 
+#### Examples
+
+- 1:1 relationship
+
+```javascript
+// users like { _id: '111', name: 'John', roleId: '555' }
+// roles like { _id: '555', permissions: ['foo', bar'] }
+import { populate } from 'feathers-hooks-common';
+
+const userRoleSchema = {
+  include: {
+    service: roles,
```

Comment:
The examples as currently written will fail, as the values are missing single quotes.